src/components/StatusDot/StatusDot.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import { getRadii, pxToRem } from '../../utils';
+
+const StatusDotColors = {
+  good: '100, 244, 81',
+  bad: '242, 46, 67',
+  neutral: '84, 129, 217',
+  inactive: '216, 216, 216',
+} as const;
+
+const StatusDotComponent = Styled.div`
+  width: ${pxToRem(12)};
+  height: ${pxToRem(12)};
+  border-radius: ${getRadii('round')};
+  background-color: rgb(${({ color }) => StatusDotColors[color]});
+  border: 2px solid white;
+  box-shadow: 0 0 12px 0px rgba(${({ color }) =>
+    `${StatusDotColors[color]}, 0.5`});
+`;
+
+const StatusDot = ({
+  color = 'inactive',
```

Comment:
should we change the prop name to sentiment instead of color? since the values are not colors

Reply:
I personally wouldn't change the prop name, although I understand your point. `Color` is more predictable name than `sentiment` and values will be autocomplete by consumer IDE